### DISCUSSION TITLE
Somebody know where "x-robots-tag" is coming in rspress docs site?

### DISCUSSION THREAD

**Discussion by ArnavK-09** [web-infra-dev/rspress#628] [Q&A] [answered]
![Screenshot_2024-02-13-22-28-22-84_40deb401b9ffe8e1df2f1cc5ba480b12.jpg](https://github.com/web-infra-dev/rspress/assets/69188140/a01a1b6d-67a8-4fa0-82b6-39bde98fb747)

**Comment by xc2** [comment, marked as answer]
Vercel add this header for preview deploymenets by default to avoid the preview deployments indexed by search engines.

**Reply by ArnavK-09** [reply]
Thanks 🦇
